hack/build.sh

```diff
@@ -1,6 +1,15 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -e
-OUTFILE=/usr/local/bin/irisctl
-go build -o $OUTFILE main.go
 
-chmod +x $OUTFILE
\ No newline at end of file
+OUTFILE=${1:-/usr/local/bin/irisctl}
```

Comment:
This will default to the "local" location, but allows an override to be passed in.